### ISSUE TITLE
Feature Request - dice accounting for harm

### ISSUE THREAD

**Issue by cojoMan** [justinross/foundry-bitd-alternate-sheets#81]
So if any level 1 harm has some text (spaces not considered) - a default risky/standard would roll as risky/limited.
Any level 2 harm would take the dice down a peg - special consideration if that would mean no dice, so roll twice, take the lower.

**Comment by justinross** [comment]
Thanks for the request! This isn’t entirely possible, since (as I understand it) the harm penalties are situational, so you only suffer a penalty if the harm would reasonably hinder the action you’re rolling for, rather than a blanket penalty to all rolls.

e.g., a level 1 harm in your leg isn’t really going to give you a penalty on a roll to bluff your opponent in a game of cards.

Anyway, the closest I could probably do is add something to the roll pop asking if you want to take harm into account (if present). 

I’ll look into it!

**Comment by cojoMan** [comment]
Oh poopp, yes that's perfectly reasonable, more so that the harm could be physical and not apply to a "mental" roll, or the other way around. 
Might I suggest having an instance-copy of the harm filled droplets from the sheet on the die roll screen, or better yet, have it drop as a message on a roll that has just happened.
eg. PlayerA rolls a risky standard, gets a partial success with a 4, and that roll has the message "harms : level1/level3", or listing the harms...or something like that...

**Comment by cojoMan** [comment]
Oh poopp, yes that's perfectly reasonable, more so that the harm could be physical and not apply to a "mental" roll, or the other way around. 
Might I suggest having an instance-copy of the harm filled droplets from the sheet on the die roll screen, or better yet, have it drop as a message on a roll that has just happened.
eg. PlayerA rolls a risky standard, gets a partial success with a 4, and that roll has the message "harms : level1/level3", or listing the harms...or something like that...

**Comment by justinross** [comment]
Yeah, a reminder of harm on the dice popup is a good idea. I'd have to look into implementation, though, as the dice roller isn't something I'm currently customizing (I think? It's been awhile).
